Fix frontmatter table and use comma for restrictive modifier (#175)
grafana/writers-toolkit: docs/sources/writing-guide/front-matter/index.md

      - architecture
    ---

The following list describes what each element does and provides guidelines for its content.

`title` **Required.**

:  Becomes the document title element. Often browsers display this in the tab for the page.

   It doesn't need to precisely match the `menuTitle`.

>    The title should be optimized for search engines. May be surrounded by double quote marks ("). Do not use smart quotes.

Reviewer: ```suggestion
   Optimize the title for search engines. Use double quotes (`"`) to surround the title. Do not use smart quotes.
```

`description`

> :  Displays as a clue to users about what the page should include on social (Twitter and the like),

Reviewer: ```suggestion
:  Displays as a clue to users about what the page includes on social, such as Twitter,
```

> :  Displays as a clue to users about what the page should include on social (Twitter and the like),
>    though not as much by a search engine.

Reviewer: ```suggestion
:  On social media, such as Twitter, displays as a clue to users about what the page includes.
```

>    The number of characters vary by media, but use them wisely.

Reviewer: ```suggestion
   The number of characters vary by media, so make the description concise.
```

>    Provide enough information to guide users to the content by describing what content is provided using the link.

Reviewer: ```suggestion
   Provide enough information to guide users to the content by describing what content and include the link.
```
I am having a hard time parsing this sentence.

>    Often, this doesn’t need to be original prose&mdash;you can often scan the first few paragraphs to pluck the appropriate terms or phrases into the description.

Reviewer: I like the word "pluck" here, so I am leaving it. :) "Ppypph!"

>    It won't cause harm if it's too long, it will simply truncate in the displayed media.

Reviewer: ```suggestion
   If it's too long, it is harmlessly truncated on social media.
```

>    May be surrounded by double quote marks ("). Do not use smart quotes.

Reviewer: ```suggestion
   Use double quotes (`"`) to surround the title. Do not use smart quotes.
```
Writing a happy path. Less thinking if we say, "Use ..." instead of "You can use ..."

`aliases`

:  Provides an HTML redirect from the pages in the list to the current page.
   Described in detail in Hugo aliases.

`weight`

> :  Determines the placement of the topic within the left-hand sidebar of our website, with smaller numbers placing the topic higher in the guide. Pages with the same weight have lexicographic ordering.

Reviewer: ```suggestion
:  Determines the placement of the topic within the left-hand sidebar on https://grafana.com. Smaller numbers place the topic higher in the guide. Pages with the same weight have lexicographic ordering.
```

Reviewer: I removed the possessive "our website" because it is unnecessary possession.

>    Use increments of `100` for all other content files, because doing so eliminates much of the need to re-order existing topics when new topics are added. Weights are per directory.

Reviewer: ```suggestion
   Use increments of `100` for all other content files. Doing so makes it easier for you to re-order existing topics when you add new topics. Weights are per directory.
```

`keywords`

> :  Keywords are used by the website to link to related pages in the “related content” sections.

Reviewer: ```suggestion
:  The website uses keywords to link to related pages in the _Related content_ sections.
```
Rewording to remove the agent participle.

>    They do not appear in the resulting HTML source for the page and have no effect on SEO.

Reviewer: ```suggestion
   They do not appear in the resulting HTML source for the page and do not affect SEO.
```

Reviewer: Nit about parallel sentence structure.
